docs/data/install-xrootd.md

```diff
@@ -290,6 +290,70 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 **Note**: In this example, `fermicloud53.fnal.gov` is a redirector node and `fermicloud054.fnal.gov` is a data node.
 
+### (Optional) Enabling Xrootd over HTTP
+
+XRootD can be accessed using the HTTP protocol. To do that:
+
+1.   Modify `/etc/xrootd/xrootd-clustered.cfg` and add the following lines:
+
+        :::file
+           sec.protocol /usr/lib64 gsi \
+              -certdir:/etc/grid-security/certificates \
+              -cert:/etc/grid-security/xrd/xrdcert.pem \
+              -key:/etc/grid-security/xrd/xrdkey.pem \
+              -crl:3 \
+              -authzfun:libXrdLcmaps.so \
+              -authzfunparms:--lcmapscfg,/etc/xrootd/lcmaps.cfg,--loglevel,4 \
+              -gmapopt:10 \
+              -gmapto:0
+           if exec xrootd
+            xrd.protocol http:1094 libXrdHttp.so
+            http.cadir /etc/grid-security/certificates
+            http.cert /etc/grid-security/xrd/xrdcert.pem
+            http.key /etc/grid-security/xrd/xrdkey.pem
+            http.secxtractor /usr/lib64/libXrdLcmaps.so
+            http.listingdeny yes
+            http.staticpreload http://static/robots.txt /etc/xrootd/robots.txt
+            http.desthttps yes
+           fi
+  
+1.   Create robots.txt. Add file `/etc/xrootd/robots.txt` with these contents:
+
+        :::file
+           User-agent: *
+           Disallow: / 
+
+1.   Testing the configuration
```

Comment:
Start the service, of course!

Reply:
Good catch, I will link to the manage services